### PR TITLE
feat(frontend): update the module selection

### PR DESCRIPTION
**Pull request by kelsos**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

## Notes

I tried a couple of other approaches before this and I was not really happy.
This is actually the 3rd approach I tried.

### CODE REVIEW

File: frontend/app/src/components/defi/wizard/ModuleSelector.vue [hunk 1]
```diff
@@ -1,145 +1,163 @@
...
-        close
-        pill
-        @click:close="unselect(data.item.identifier)"
-      >
-        <v-avatar left class="d-flex">
-          <adaptive-wrapper
-            class="d-flex align-center"
-            width="100%"
-            height="100%"
+  <card v-if="true" outlined-body flat>
```
Comment (lukicenturi) [inline frontend/app/src/components/defi/wizard/ModuleSelector.vue:102]: remove the if

**Comment by lukicenturi** [comment]
I think this padding need to be removed 
image

**Comment by lukicenturi** [comment]
in dark mode, this can be improved (put some distance)
image

**Comment by kelsos** [comment]
> I think this padding need to be removed image

@lukicenturi so you noticed? damn 😃

File: frontend/app/src/components/defi/wizard/ModuleSelector.vue (hunk 1 above)
Comment (kelsos) [reply frontend/app/src/components/defi/wizard/ModuleSelector.vue:102]: A yes that was from testing the other without removing the old component, good catch.

File: frontend/app/src/components/defi/wizard/ModuleSelector.vue [hunk 2]
```diff
@@ -1,145 +1,163 @@
...
-  const callback = () => balancesStore.$reset();
-  setTimeout(callback, 800);
-};
+const modules = computed<(SupportedModule & { enabled: boolean })[]>(() => {
+  const active = get(activeModules);
+  const filter = get(search);
+  return supportedModules
+    .filter(m =>
+      filter ? m.name.toLowerCase().includes(filter.toLowerCase()) : true
+    )
```
Comment (lukicenturi) [inline frontend/app/src/components/defi/wizard/ModuleSelector.vue:40]: Is this better, no need to do the loop if the search is empty?

`(filter ? supportedModules : supportedModules.filter(...))`

**Review by lukicenturi — APPROVED** [review]

**Pull request merged**